Update Windows installer documentation for gMSA release (#14578)
DataDog/documentation: content/en/agent/guide/windows-agent-ddagent-user.md

* Deny log on through Remote Desktop Services
* Log on as a service

**Important**: Since the account is modified during installation to restrict its privileges, including login privileges, make sure it is not a 'real' user account but an account solely dedicated to run the Datadog Agent.

**Note**: All example commands on this page use `<>` to indicate a variable that should be replaced. For example, if the user account is `ddagentuser` and the command contains `DDAGENTUSER_NAME=<USERNAME>` then `DDAGENTUSER_NAME=ddagentuser` should be entered in the command line.

> **Note**: Starting with release `7.38.0/6.38.0` the installer supports **Grouped Managed Service Account (gMSA)**. To specify a Grouped Managed Service Account, append **$** at the end of the username, i.e. `<DOMAIN>\<USERNAME>$`. The Grouped Managed Service Account must exist *prior* to installation, the installer cannot it.

Reviewer: ```suggestion
**Note**: Starting with release `7.38.0/6.38.0` the installer supports the use of a **Grouped Managed Service Account (gMSA)**. To specify a Grouped Managed Service Account, append **$** at the end of the username: `<DOMAIN>\<USERNAME>$`. The Grouped Managed Service Account must exist *prior* to installation, as the installer cannot create one.
```
